.github/workflows/ci-xml-reserved-characters.yml

```
@@ -33,7 +33,7 @@ jobs:
         run: sudo apt-get install -y pandoc
 
       - name: Build Arise
-        run: bash arise build
+        run: bash arise
```

rejecting breaking change to invocation syntax
```suggestion
        run: bash arise build
```